doc/requirements.txt

```diff
@@ -5,4 +5,5 @@ graphviz
 pybind11
 sphinx
 sphinx-automodapi
-pennylane-sphinx-theme
+git+https://github.com/PennyLaneAI/pennylane.git@master
```

Comment:
Is this needed for something?
In case not, I think it is preferable to not add the pennylane installation to this file, since these are installations directly related to docs.

Reply:
Was just trying to make the `doc` CI pass.

Reply:
NP. Docs are supposed to keep failing until I merge [PR 473](https://github.com/PennyLaneAI/pennylane-lightning/pull/473).